app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,27 @@
+class InvitationsController < ApplicationController
+  def new
+    @invitation = Invitation.new
+  end
+
+  def create
+    user = User.find_by(inv_params)
+    event_id = cookies[:event_id]
+    cookies[:event_id] = nil
+    if user
+      @invitation = Invitation.new
+      @invitation.event_id = event_id
+      @invitation.attendee_id = user.id
+      @invitation.save
+      redirect_to event_path(event_id), notice: "#{user.name} was invited!"
+
+    else
+      redirect_to event_path(event_id), notice: 'Invalid user name'
+    end
+  end
+
+  private
+
+  def inv_params
+    params.permit(:name)
```

Comment:
- [ ] You are using the `name` to find the User but the name is not being validated as required to be unique. What happens if two users have the same name?
Use the id of the user instead of `name`, that you will be always sure it's unique.

- [ ] Add the event id to the inv params too. That way you won't have to use cookies.